client/src/components/ManageAccount/ManageAccount.scss

```diff
@@ -124,14 +124,17 @@
   color: white !important; /* override `.workbench-content .btn-secondary` */
 }
 
+.googledrive-submit-button-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
 .modal-body {
   font-size: 14px;
   .btn-link {
     padding: 0;
   }
-  .manage-account-submit-button {
-    margin-left: auto;
-  }
```

Comment:
__Warning__

There is markup that still uses this class (including markup you've edited in this PR).
1. All such markup locations would now have the cruft of a leftover class. But, the class and styles should remain; see point 2.
2. These buttons are now left-aligned, instead of right-aligned. But, form submit buttons should be right-aligned.

Reference:
- most, if not all all, form submit buttons in the [Adobe XD Core Portal design](https://xd.adobe.com/view/db2660cc-1011-4f26-5d31-019ce87c1fe8-ad17/grid)

Reply:
__Short Fix__
- Remove the new class from the CSS, and its alternate spelling in the markup.
- Restore the deleted class.

<details><summary><strong>Analysis of Problem</strong></summary>

I think the miscommunication and missteps were:

1. Ticket says "Sal: Google Drive Privacy Policy Modal button should be smaller in width, and have button text centered".
2. A style was intended to be added via [`googledrive-submit-button-container`](https://github.com/TACC/Core-Portal/pull/354/files#diff-d7ac78611d5cf8031657d0dedaef5e5888484c459e543e9a8772ea3363d8ac6eR127), but the class in the markup is spelled [`google-drive-submit-button-container`](https://github.com/TACC/Core-Portal/pull/354/files#diff-fcb20c4c812e715e47561e2483cf915559b647ea75c4f9eae8d5c6c2ce6bed1eR170), so the styles are not even applied.
3. The styles, if applied, would center the button within the space at the bottom of the form. This is a misinterpretation of Sal's message, which is understandable. He does not describe centering the button itself.
    - <details><summary>What did Sal mean?</summary>

        He expects the __text__ of the button to be centered with a smaller button. But a simpler description of the expectation is that the button should be sized like all the others (technically, shrink automatically to the size of the text plus the padding that exists from pre-existing styles). The alignment of the button should not change.

        </details>
4. The CSS ruleset for `manage-account-submit-button` was removed in favor of the new inaccurately applied one-off submit button class, without resolving the other affected instances.

</details>

Reply:
Good catch. I also believe this has inadvertently been fixed in another PR, as the button is appropriately sized etc. in prod:
<img width="528" alt="Screen Shot 2021-03-04 at 7 34 41 PM" src="https://user-images.githubusercontent.com/20326896/110054432-b2f8b000-7d20-11eb-978d-af94afa79ac3.png">